watertap/property_models/tests/test_multicomp_aq_sol_prop_pack.py

```diff
@@ -1121,11 +1121,8 @@ def test_parameter_block_comparison(model4):
 
     assert isinstance(m_ion.fs.properties.solute_set, Set)
     assert isinstance(m_generic.fs.properties.solute_set, Set)
-    assert len(m_ion.fs.properties.solute_set) == len(
-        m_generic.fs.properties.solute_set
-    )
```

Comment:
Ah, the intention of this test was to verify compatibility between this prop package and the (formerly known as) generic property framework in IDAES. I am guessing you had to remove this since you redefined how the sets are organized, and so things might not line up anymore with the generic property framework.

Reply:
That is correct.  My key points on this are (1) I didn't see the point that we want a property package interchangeable to an existing one, (2) idaes's generic p.p. does not use and is inconsistent to its `component.py` module, where `Solute` is the parent class of `Ion`, (3) it's a basic knowledge that for the solutes of a solution can embrace ionic species if they are solvated.  Therefore, I am in favor of making this change, although I will also be ok with not making the change and disregarding this PR.